indico/htdocs/js/indico/RegistrationForm/section.js

```diff
@@ -705,8 +705,9 @@ ndRegForm.directive("ndSocialEventSection", function() {
                            align: 'center',
                            width: 50,
                            editable: true,
-                           edittype: "text",
+                           edittype: 'text',
                            editoptions: {size: "7", maxlength: "20"},
+                           pattern: '/^(\\d*[.])?\\d+$/'
```

Comment:
I think this regex would make more sense:

    ^(\\d+(\\.\\d{2})?)?$

For monetary amounts only 2 digits behind the decimal point make sense.